### PR TITLE
Bug 1137713 - Fix task manager transitions from/to homescreen. r=alive

### PR DESCRIPTION
**Pull request by Cwiiis**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=18fe5c1a32785049253eca2d679381554422a774

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=48a0180a1247d43fad829166373fbdea7867c034

File: apps/system/style/window.css [hunk 1]
```diff
@@ -538,7 +556,9 @@
 
 /* making the appWindows moz-elementable for the cardview */
 #screen #windows > .appWindow:not(.homescreen):not(.lockScreenWindow):not(.attentionWindow):not(.lockScreenInputWindow).in-task-manager {
+  position: fixed;
   transform: none;
+  z-index: -1;
```
Comment (alivedise) [inline apps/system/style/window.css]: put this in z-index.css

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ae073ae81f8bb056b92f248ef41b409b7d7d2e11

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#pyz973jJQf-lZrsh8Toriw

The pull request failed to pass integration tests. It could not be landed, please try again.

**Pull request merged**